toolchain/sem_ir/typed_insts.h

```diff
@@ -1461,6 +1461,13 @@ struct VtableType {
   TypeId type_id;
 };
 
+// Initializer for virtual function table pointers in object initialization.
+struct VptrInit {
```

Comment:
Yep, that'd be a totally reasonable name for this, I think. Happy to leave the naming question open while the rest of this is reviewed, and change to whatever it is folks prefer towards the end, to reduce churn in the review if the naming continues to be considered.

(broader rambling: I am starting to get more of a feel for all this, and might be approaching the point where I could write up some thoughts on how this should look overall, from calls to initialization, to handling vtable decls/defs/etc - to get a better idea of what should be encoded in SemIR and what should be hidden and purely an artifact of lowering... (like are we going to need some other new instruction for virtual calls, how do we differentiate declaration and definition of the vtable (does that need to be in SemIR at all, or can/should we hide it in lowering, etc), etc))

Reply:
> Yep, that'd be a totally reasonable name for this, I think.

That'd be my preference then, because I find it more concrete and intuitive.

> Happy to leave the naming question open while the rest of this is reviewed, and change to whatever it is folks prefer towards the end, to reduce churn in the review if the naming continues to be considered.

I'm not sure if you're expecting a review from anybody else, but FWIW I have no other comments.

Reply:
Oh, OK - no particular expectation. Happy to move forward with the rename then. (was sort of expecting this might be more contentious/debateable, but not objecting if it isn't :) )

Reply:
Done in 186c21ca9a7e53ec19360020a02221077dc277ed